src/praisonai-bot/praisonai_bot/scheduler/executor.py

```
@@ -0,0 +1,598 @@
+            )
+            if supports_session_id:
+                if session_target == "main" and session_id:
+                    chat_kwargs["session_id"] = session_id
+                else:
+                    chat_kwargs["session_id"] = f"cron_{job.id}"
+
+            result = await asyncio.to_thread(
+                agent.chat, message, **chat_kwargs,
+            )
+            result_str = str(result)
+        except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)